tests/accessibility/.pa11y-ci.js

```diff
@@ -7,6 +7,11 @@ const {
   reviewNHSAppTemplatePage,
   reviewNHSAppTemplateErrorPage,
   submitNHSAppTemplatePage,
+  createEmailTemplatePage,
+  createEmailTemplateErrorPage,
+  createTextMessageTemplatePage,
+  createTextMessageTemplateErrorPage,
+  NHSAppTemplateSubmittedPage,
```

Comment:
In code you don't need to maintain proper noun capitalisation, i.e. this would be nicer as `nhsAppTemplateSubmittedPage`

Reply:
captured in tidy up task https://nhsd-jira.digital.nhs.uk/browse/CCM-6950